app/models/mise_en_action.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class MiseEnAction < ApplicationRecord
+  belongs_to :evaluation
+
+  validates :evaluation_id, uniqueness: true
+
+  after_update :enregistre_date, if: :saved_change_to_effectuee?
+
+  acts_as_paranoid
+
+  def enregistre_date
+    update(repondue_le: Time.zone.now)
+  end
```

Comment:
J'utiliserai le `before_save` plutot car on assigne `effectuee` à la création.

J'utilise un before plutot qu'on after afin de définir l'attribut en amont et refaire une requete SQL

```suggestion
  before_save :enregistre_date, if: :saved_change_to_effectuee?

  acts_as_paranoid

  def enregistre_date
    self.repondue_le = Time.zone.now
  end
```